patterns/vpc-lattice/cross-cluster-pod-communication/cluster/charts/demo/templates/app-deploy.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-{{ .Values.version }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ .Release.Name }}-{{ .Values.version }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}-{{ .Values.version }}
+  template:
+    metadata:
+      annotations:
+       vpc-lattices-svcs.amazonaws.com/agent-inject: "true"
+      labels:
+        app: {{ .Release.Name }}-{{ .Values.version }}
+    spec:
+      containers:
+      - name: {{ .Release.Name }}-{{ .Values.version }}
+        image: public.ecr.aws/seb-demo/http-server:latest
```

Comment:
Can we use some public image like [this](https://github.com/kubernetes-sigs/aws-load-balancer-controller/blame/e5d625f96415fd44e6399e9c75e2bd985f5a2288/docs/examples/echoservice/echoserver-deployment.yaml#L17) instead?

Reply:
not sure, the code to build the container is public : https://github.com/aws-samples/amazon-eks-security-immersion-day/blob/mainline/docker/http-server/launch_app.sh and I use a script to configure the CA at runtime

Reply:
I put the link to the Dockerfile used